pyEpiabm/pyEpiabm/routine/household_infection.py

```diff
@@ -0,0 +1,57 @@
+#
+# Calculate household force of infection based on Covidsim code
+#
+
+from pyEpiabm.core import Person
+
+
+class HouseholdInfection:
+    """Class to calculate the infectiousness and susceptibility
+    parameters for the force of infection parameter, within households.
+    """
+    @staticmethod
+    def household_inf(infector: Person, timestep: int):
+        """Calculate the infectiveness of a household.
```

Comment:
```suggestion
        """Calculate the infectiousness of a household.
```
Saves us from having to make extra changes on the other PR